### PR TITLE
[refactor] Use a more descriptive method name

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/services/folio/types.rb [hunk 1]
```diff
@@ -46,42 +46,42 @@ def circulation_rules
...
+      @libraries ||= load_cache(type: 'libraries').index_by { |p| p['id'] }
     end
 
     def locations
-      @locations ||= get_type('locations').index_by { |p| p['id'] }
+      @locations ||= load_cache(type: 'locations').index_by { |p| p['id'] }
     end
 
-    def get_type(type)
+    def load_cache(type:)
```
Comment (cbeer) [inline app/services/folio/types.rb:84]: I personally like the current name more than `load_cache`. I also don't know why we want a keyword arg here.

**Review by marlo-longley — APPROVED** [review]

File: app/services/folio/types.rb (hunk 1 above)
Comment (marlo-longley) [reply app/services/folio/types.rb:84]: Oops. I thought it looked OK but happy to defer / discuss.

Comment (jcoyne) [reply app/services/folio/types.rb:84]: `load_cache` tells me what the method is doing. It's populating a cache from the files on disk.  `get_type` on the other hand is two very ambigous words. "get" from where?  "type" of what?

Comment (cbeer) [reply app/services/folio/types.rb:84]: I don't think being cached is the important part of this. We're just trying to load FOLIO type data.

**Pull request closed without merging**